code/game/gamemodes/cult/runes.dm

```diff
@@ -452,6 +455,30 @@ var/list/teleport_runes = list()
 	scribe_damage = 40.1 //how much damage you take doing it
 	var/used
 
+/obj/effect/rune/narsie/New()
+	. = ..()
+	SSobj.processing += src
+	poi_list |= src
+
+/obj/effect/rune/narsie/Destroy()
+	SSobj.processing -= src
+	poi_list -= src
+	. = ..()
+
+/obj/effect/rune/narsie/process()
+	// Automatically attempt a summoning, once per process
+	var/list/invokers = can_invoke()
+	if(invokers.len >= req_cultists)
+		invoke(invokers)
+	else
+		fail_invoke(message=FALSE)
+	// Automatically disappear if for some reason it's not present on station
+	if(loc.z != ZLEVEL_STATION)
```

Comment:
also this should be ABOVE the 'attempt invoke' or people very well WILL manage to summon off-station